### ISSUE TITLE
Azure: blobstore missing in bosh yaml

### ISSUE THREAD

**Issue by haydonryan** [enaml-ops/omg-cli#11]
More template issues from command:

```

./omg-linux azure \
  --azure-vnet boshvnet-crp \
  --azure-subnet Bosh \
  --azure-subscription-id <Redacted> \
  --azure-tenant-id <Redacted>  \
  --azure-client-id <Redacted> \
  --azure-client-secret amazon4ever \
  --azure-resource-group hrcentral \
  --azure-storage-account <Redacted> \
  --azure-security-group nsg-bosh \
  --azure-ssh-pub-key-path bosh-privatekey.pub \
  --azure-ssh-user vcap \
  --azure-environment AzureCloud \
  --azure-private-key-path bosh-privatekey \
  --cpi-name cpi \
  --bosh-private-ip 10.0.0.5 \
  --print-manifest

```

```
Uploading stemcell 'bosh-azure-hyperv-ubuntu-trusty-go_agent/3232.8'... Skipped [Stemcell already uploaded] (00:00:00)

Started deploying
  Waiting for the agent on VM '55h755votbjp2cfdefaultsa-e4416f6a-9a10-4ae0-6b03-749e145aa955'... Finished (00:00:00)
  Stopping jobs on instance 'unknown/0'... Finished (00:00:00)
  Unmounting disk 'bosh-data-55h755votbjp2cfdefaultsa-41580660-ef72-45ea-96bc-c3a2bf5acd81-None'... Finished (00:00:01)
  Deleting VM '55h755votbjp2cfdefaultsa-e4416f6a-9a10-4ae0-6b03-749e145aa955'... Finished (00:02:40)
  Creating VM for instance 'bosh/0' from stemcell 'bosh-stemcell-0a915ed0-638d-4171-949a-e4b6c719666a'... Finished (00:01:55)
  Waiting for the agent on VM '55h755votbjp2cfdefaultsa-df5639a3-edf1-45af-6d87-bcb5f5a08505' to be ready... Finished (00:00:21)
  Attaching disk 'bosh-data-55h755votbjp2cfdefaultsa-41580660-ef72-45ea-96bc-c3a2bf5acd81-None' to VM '55h755votbjp2cfdefaultsa-df5639a3-edf1-45af-6d87-bcb5f5a08505'... Finished (00:00:45)
  Rendering job templates... Failed (00:00:01)
Failed deploying (00:05:48)

Stopping registry... Finished (00:00:00)
Cleaning up rendered CPI jobs... Finished (00:00:00)

Command 'deploy' failed:
  Deploying:
    Building state for instance 'bosh/0':
      Rendering job templates for instance 'bosh/0':
        Rendering templates for job 'director/8b7977b57a31aea379d10ab04851d0ea57009b07':
          Rendering template src: director.yml.erb.erb, dst: config/director.yml.erb:
            Rendering template src: /home/haydon/.bosh_init/installations/e641e384-5322-4095-4fba-6c5c53a35dbd/tmp/bosh-init-release672283891/extracted_jobs/director/templates/director.yml.erb.erb, dst: /home/haydon/.bosh_init/installations/e641e384-5322-4095-4fba-6c5c53a35dbd/tmp/rendered-jobs839132171/config/director.yml.erb:
              Running ruby to render templates:
                Running command: 'ruby /home/haydon/.bosh_init/installations/e641e384-5322-4095-4fba-6c5c53a35dbd/tmp/erb-renderer602830570/erb-render.rb /home/haydon/.bosh_init/installations/e641e384-5322-4095-4fba-6c5c53a35dbd/tmp/erb-renderer602830570/erb-context.json /home/haydon/.bosh_init/installations/e641e384-5322-4095-4fba-6c5c53a35dbd/tmp/bosh-init-release672283891/extracted_jobs/director/templates/director.yml.erb.erb /home/haydon/.bosh_init/installations/e641e384-5322-4095-4fba-6c5c53a35dbd/tmp/rendered-jobs839132171/config/director.yml.erb', stdout: '', stderr: '/home/haydon/.bosh_init/installations/e641e384-5322-4095-4fba-6c5c53a35dbd/tmp/erb-renderer602830570/erb-render.rb:180:in `rescue in render': Error filling in template '/home/haydon/.bosh_init/installations/e641e384-5322-4095-4fba-6c5c53a35dbd/tmp/bosh-init-release672283891/extracted_jobs/director/templates/director.yml.erb.erb' for director/0 (line 140: #<TemplateEvaluationContext::UnknownProperty: Can't find property 'blobstore.address'>) (RuntimeError)
    from /home/haydon/.bosh_init/installations/e641e384-5322-4095-4fba-6c5c53a35dbd/tmp/erb-renderer602830570/erb-render.rb:166:in `render'
    from /home/haydon/.bosh_init/installations/e641e384-5322-4095-4fba-6c5c53a35dbd/tmp/erb-renderer602830570/erb-render.rb:191:in `<main>'
':
                  exit status 1
```

**Comment by calebwashburn** [comment]
This property is hardcoded to 127.0.0.1 in latest version.  Can you include a redacted --print-manifest

**Comment by haydonryan** [comment]
```
name: enaml-bosh
releases:
- name: bosh
  url: https://bosh.io/d/github.com/cloudfoundry/bosh?v=256.2
  sha1: ff2f4e16e02f66b31c595196052a809100cfd5a8
- name: bosh-azure-cpi
  url: https://bosh.io/d/github.com/cloudfoundry-incubator/bosh-azure-cpi-release?v=11
  sha1: 395fc05c11ead59711188ebd0a684842a03dc93d
networks:
- name: private
  type: manual
  subnets:
  - range: 10.0.0.0/24
    gateway: 10.0.0.1
    dns:
    - 168.63.129.16
    cloud_properties:
      virtual_network_name: boshvnet-crp
      subnet_name: Bosh
- name: public
  type: vip
resource_pools:
- name: vms
  network: private
  stemcell:
    url: https://bosh.io/d/stemcells/bosh-azure-hyperv-ubuntu-trusty-go_agent?v=3169
    sha1: ff13c47ac7ce121dee6153c1564bd8965edf9f59
  cloud_properties:
    instance_type: Standard_D1
disk_pools:
- name: disks
  disk_size: 20000
  cloud_properties: null
jobs:
- name: bosh
  templates: [{name: nats, release: bosh}, {name: postgres, release: bosh}, {name: registry,
      release: bosh}, {name: director, release: bosh}, {name: blobstore, release: bosh},
    {name: health_monitor, release: bosh}, {name: cpi, release: bosh-azure-cpi}]
  persistent_disk_pool: disks
  properties:
    agent:
      mbus: nats://nats:nats-password@10.0.0.5:4222
    azure:
      ssh_user: vcap
      resource_group_name: hrcentral
      client_secret: amazon4ever
      storage_account_name: 55h755votbjp2cfdefaultsa
      ssh_public_key: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDXmChgKAeewoLzGUuFasX51qEfo7FWvuUo/WmQA6lmRU5wrWwwzhtBbaDrt2M+Nlb4QMw8oyrd+hbCj0rPUt+KXAoazkcP2Uzc+fI2892Jhs4UniPDjbCFqT6i716rZ50pm7ufaCccM8pV+wJJVnGQIFvq2QfqhQq66zp4f0qPaNjgyGrDCK1Lx7fjSYBx13/P9lQsXAsV1akrsdRCsq2fs8Mi28OJ0+56coSUqrYy5peZUP+pMo6SPjx9v2D1aD0JQitZ85t615HPK0UCd1gw+f6hY+aBC8wPnR+d+mun+qK/wAiP6pyTEtxiqwOd+dikJWzTa9PQ++ja1RmgpxWp
        \n"
      environment: AzureCloud
      default_security_group: nsg-bosh
      client_id: e9d67750-01ab-45ea-908d-fa724cef9f8d
      subscription_id: 84ffb17d-0826-4946-b6cf-1311e401a22c
      tenant_id: 1fb8ec91-a6f7-4b90-b7b1-3efb5523fd34
    blobstore:
      port: 25250
      director:
        password: pi02rv4svde7h05ypq4q
        user: director
      agent:
        user: agent
        password: c2d9z8qisgzwaih9vd2q
    director:
      cpi_job: cpi
      name: enaml-bosh
      max_threads: 10
      db:
        user: postgres
        host: 127.0.0.1
        password: bi9rd3fiy3u0xbw2eqc0
        port: 5432
        adapter: postgres
      user_management:
        local:
          users:
          - name: director
            password: pi02rv4svde7h05ypq4q
          - name: hm
            password: r2gyqtuftpy7pf6e0s95
        provider: local
    hm:
      resurrector: {}
      resurrector_enabled: true
      director_account:
        user: hm
        password: r2gyqtuftpy7pf6e0s95
    nats:
      user: nats
      listen_address: 127.0.0.1
      password: c2d9z8qisgzwaih9vd2q
    ntp:
    - 0.pool.ntp.org
    - 1.pool.ntp.org
    postgres:
      listen_address: 127.0.0.1
      database: bosh
      additional_databases:
      - uaa
      - registry
      user: postgres
      password: bi9rd3fiy3u0xbw2eqc0
    registry:
      http:
        user: admin
        password: ygpvsvxy4eyutvr6wpci
        port: 25777
      db:
        port: 5432
        password: bi9rd3fiy3u0xbw2eqc0
        database: registry
        adapter: postgres
        user: postgres
  resource_pool: vms
  instances: 1
  networks:
  - name: private
    static_ips:
    - 10.0.0.5
    default:
    - dns
    - gateway
  - name: public
    static_ips:
    - 10.0.0.5
cloud_provider:
  template: {name: cpi, release: bosh-azure-cpi}
  mbus: https://mbus:mbus-password@10.0.0.5:6868
  properties:
    agent:
      mbus: https://mbus:mbus-password@0.0.0.0:6868
    azure:
      ssh_user: vcap
      resource_group_name: hrcentral
      client_secret: NO
      storage_account_name: NO
      ssh_public_key: ""
      environment: AzureCloud
      default_security_group: nsg-bosh
      client_id: NO
      subscription_id: NO
      tenant_id: NO
    blobstore:
      path: /var/vcap/micro_bosh/data/cache
      provider: local
    ntp:
    - 0.pool.ntp.org
    - 1.pool.ntp.org
  ssh_tunnel:
    host: 10.0.0.5
    port: 22
    user: vcap
    private_key: bosh-privatekey
```

**Comment by haydonryan** [comment]
```
name: enaml-bosh
releases:
- name: bosh
  url: https://bosh.io/d/github.com/cloudfoundry/bosh?v=256.2
  sha1: ff2f4e16e02f66b31c595196052a809100cfd5a8
- name: bosh-azure-cpi
  url: https://bosh.io/d/github.com/cloudfoundry-incubator/bosh-azure-cpi-release?v=11
  sha1: 395fc05c11ead59711188ebd0a684842a03dc93d
networks:
- name: private
  type: manual
  subnets:
  - range: 10.0.0.0/24
    gateway: 10.0.0.1
    dns:
    - 168.63.129.16
    cloud_properties:
      virtual_network_name: boshvnet-crp
      subnet_name: Bosh
- name: public
  type: vip
resource_pools:
- name: vms
  network: private
  stemcell:
    url: https://bosh.io/d/stemcells/bosh-azure-hyperv-ubuntu-trusty-go_agent?v=3169
    sha1: ff13c47ac7ce121dee6153c1564bd8965edf9f59
  cloud_properties:
    instance_type: Standard_D1
disk_pools:
- name: disks
  disk_size: 20000
  cloud_properties: null
jobs:
- name: bosh
  templates: [{name: nats, release: bosh}, {name: postgres, release: bosh}, {name: registry,
      release: bosh}, {name: director, release: bosh}, {name: blobstore, release: bosh},
    {name: health_monitor, release: bosh}, {name: cpi, release: bosh-azure-cpi}]
  persistent_disk_pool: disks
  properties:
    agent:
      mbus: nats://nats:nats-password@10.0.0.5:4222
    azure:
      ssh_user: vcap
      resource_group_name: hrcentral
      client_secret: NO
      storage_account_name: NO
      ssh_public_key: "
      environment: AzureCloud
      default_security_group: nsg-bosh
      client_id: NO
      subscription_id: NO
      tenant_id: NO
    blobstore:
      port: 25250
      director:
        password: pi02rv4svde7h05ypq4q
        user: director
      agent:
        user: agent
        password: c2d9z8qisgzwaih9vd2q
    director:
      cpi_job: cpi
      name: enaml-bosh
      max_threads: 10
      db:
        user: postgres
        host: 127.0.0.1
        password: bi9rd3fiy3u0xbw2eqc0
        port: 5432
        adapter: postgres
      user_management:
        local:
          users:
          - name: director
            password: pi02rv4svde7h05ypq4q
          - name: hm
            password: r2gyqtuftpy7pf6e0s95
        provider: local
    hm:
      resurrector: {}
      resurrector_enabled: true
      director_account:
        user: hm
        password: r2gyqtuftpy7pf6e0s95
    nats:
      user: nats
      listen_address: 127.0.0.1
      password: c2d9z8qisgzwaih9vd2q
    ntp:
    - 0.pool.ntp.org
    - 1.pool.ntp.org
    postgres:
      listen_address: 127.0.0.1
      database: bosh
      additional_databases:
      - uaa
      - registry
      user: postgres
      password: bi9rd3fiy3u0xbw2eqc0
    registry:
      http:
        user: admin
        password: ygpvsvxy4eyutvr6wpci
        port: 25777
      db:
        port: 5432
        password: bi9rd3fiy3u0xbw2eqc0
        database: registry
        adapter: postgres
        user: postgres
  resource_pool: vms
  instances: 1
  networks:
  - name: private
    static_ips:
    - 10.0.0.5
    default:
    - dns
    - gateway
  - name: public
    static_ips:
    - 10.0.0.5
cloud_provider:
  template: {name: cpi, release: bosh-azure-cpi}
  mbus: https://mbus:mbus-password@10.0.0.5:6868
  properties:
    agent:
      mbus: https://mbus:mbus-password@0.0.0.0:6868
    azure:
      ssh_user: vcap
      resource_group_name: hrcentral
      client_secret: NO
      storage_account_name: NO
      ssh_public_key: ""
      environment: AzureCloud
      default_security_group: nsg-bosh
      client_id: NO
      subscription_id: NO
      tenant_id: NO
    blobstore:
      path: /var/vcap/micro_bosh/data/cache
      provider: local
    ntp:
    - 0.pool.ntp.org
    - 1.pool.ntp.org
  ssh_tunnel:
    host: 10.0.0.5
    port: 22
    user: vcap
    private_key: bosh-privatekey
```

**Comment by haydonryan** [comment]
note - passwords included are auto-regenerated anyways

**Comment by calebwashburn** [comment]
Fixed with release v0.0.10
